packages/hash/frontend/src/pages/index.page.tsx

```diff
@@ -1,44 +1,23 @@
 import { useEffect } from "react";
 import { useRouter } from "next/router";
 
-import { Box, Container, Typography } from "@mui/material";
-import { useLogoutFlow } from "../components/hooks/useLogoutFlow";
+import { Container, Typography } from "@mui/material";
 import { useLoggedInUser } from "../components/hooks/useAuthenticatedUser";
 import { NextPageWithLayout } from "../shared/layout";
-import { Button } from "../shared/ui";
 
 const Page: NextPageWithLayout = () => {
   const router = useRouter();
-  const { authenticatedUser, kratosSession } = useLoggedInUser();
-
-  const { logout } = useLogoutFlow();
+  const { authenticatedUser, loading, kratosSession } = useLoggedInUser();
 
   useEffect(() => {
     if (authenticatedUser) {
       void router.push(`/${authenticatedUser.userAccountId}`);
     }
-  }, [router, authenticatedUser]);
+  }, [router, authenticatedUser, kratosSession, loading]);
 
-  /** @todo: remove session developer information */
   return (
     <Container sx={{ pt: 10 }}>
-      {authenticatedUser && (
-        <Typography variant="h1">
-          Hi {authenticatedUser.preferredName}!
-        </Typography>
-      )}
-      <Typography variant="h2" gutterBottom>
-        {kratosSession
-          ? "You have a kratos session"
-          : "You don't have a kratos session"}
-      </Typography>
-      <Button onClick={logout}>Log out</Button>
-      <Typography gutterBottom>
-        This is what your kratos session looks like:
-      </Typography>
-      {kratosSession ? (
-        <Box component="pre">{JSON.stringify(kratosSession, null, 2)}</Box>
-      ) : null}
+      <Typography variant="h1">Homepage</Typography>
```

Comment:
```suggestion
      <Typography variant="h1">Homepage</Typography>
```
@nonparibus given that users shouldn't ever be on this page, would you be okay with just making it a blank page (rather than having "Homepage" be briefly displayed)?

Reply:
Yes. I think there was some miscommunication around the circumstances in which this would appear, originally.

Reply:
I've removed it in https://github.com/hashintel/hash/pull/1529/commits/8796d88f8ac77842469ccf9054d8c8074d648909